import the named library, and not the bare symlink that is only in -dev
--
diff --git a/bindings/python/lilv.py b/bindings/python/lilv.py
index ab1ff75..524fab9 100644
--- a/bindings/python/lilv.py
+++ b/bindings/python/lilv.py
@@ -32,7 +32,7 @@ class _LilvLib:
         elif sys.platform == "win32":
             self.lib = CDLL("lilv-0.dll")
         else:
-            self.lib = CDLL("liblilv-0.so")
+            self.lib = CDLL("liblilv-0.so.0")
 
 
 # Load lilv C library and define library global (which is populated below)
